pkgs/applications/blockchains/feather-wallet/default.nix

```diff
@@ -0,0 +1,84 @@
+{ lib, stdenv, wrapQtAppsHook, makeDesktopItem
+, fetchFromGitHub, fetchgit
+, git, cmake, qttools, pkg-config
+, qtbase, qtwebsockets, qtsvg
+, miniupnpc, unbound, readline
+, boost, libunwind, libsodium, pcsclite
+, randomx, zeromq, libgcrypt, libgpg-error
+, hidapi, rapidjson, qrencode, zbar, expat
+, libzip, libusb1, protobuf, python3
+}:
+
+stdenv.mkDerivation rec {
+  pname = "feather-wallet";
+  version = "2.1.0";
+
+  src = fetchgit {
+    url = "https://github.com/feather-wallet/feather.git";
+    rev    = "${version}";
+    sha256 = "gPNDd4hRwRCAQcvF3B+LG3tYFBkvFl5uIuF50CLcPm8=";
+    fetchSubmodules = true;
+    leaveDotGit = true;
+  };
+
+  nativeBuildInputs = [
+    cmake pkg-config wrapQtAppsHook
+    (lib.getDev qttools)
+  ];
+
+  buildInputs = [
+    qtbase qtwebsockets qtsvg
+    miniupnpc unbound readline
+    randomx libgcrypt libgpg-error
+    boost libunwind libsodium pcsclite
+    zeromq hidapi rapidjson qrencode zbar expat
+    libusb1 protobuf libzip git
+  ];
+
+  patches = [
+    ./config_fix.patch
```

Comment:
A small comment here on why this patch is needed would be helpful :)

Reply:
Comments added.